### DISCUSSION TITLE
Stand-by state switching

### DISCUSSION THREAD

**Discussion by rsaemann** [nathanmarlor/foxess_modbus#627] [Ideas]
I have a H3 inverter, which got a new feature since spring 2024: Enabeling a standby-state and setting a stand-by threashold.
This means: If the inverter detects very few energy consumption, it will switch off its inner inverter mechanics and has about 0W loss.
This is usefull in winter: My inverter has almost 150-200W power loss when switched on. So at night, it consumes more power doing nothing than it collected during the day. By using the standby-state threashhold, the inverter will put itself into idle mode when the battery is low, so I do not need buy power for it.
But in spring and summer the problem is on the other side. When the inverter detects too low power usage, he is switched off and I need to buy from the energy supplier, although my battery is almost full.

Now comes my idea for intelligent standby-state:
I would like to know if somebody has already found the addresses for this state (enabeling standby, and setting the threashold).
If battery is full, then disable the standby. Should power everything from the battery.
If battery is low, then enable standby. Go to idle state to save power comsumption, when sun is down and battery is empty.

For your interest: When in Standby/Idle state, the backup energy supply is feed directly from the grid.

**Comment by robbionline** [comment]
@rsaemann Did you find any solutions?

**Reply by MartB** [reply]
You can write to the Sys On/Off register, this should idle the inverter. 

For H3 its 41014 0: off 1: on.

**Reply by robbionline** [reply]
Thanks. I have two more questions:
- Why does FoxESS support recommend activating the hot standby?
- Which registers do I have to write for my H3-pro?


**Their Mail:**
Hello,

We currently recommend activating the hot standby feature in winter. The function you mentioned for the intelligent distinction between day and night is not yet available.


Thank you for your suggestion!


Thank you for contacting FoxESS!

With kind regards

Fox team.

**My Mail:**
My H3-Pro-30.0-E has a 200W hot standby consumption (~4.8 kWh/day). Why doesn't the inverter switch to cold standby (18W) at night after the battery reaches its minimum SoC? This causes significant losses. Is this behavior by design? Is there a setting or a planned firmware update to address this?

**Comment by rsaemann** [comment]
Hi,
 I did not yet try to write to the registers, yet. My current solution is to activate Standby state with a very low powerlimit of 20W. 
With this parameter the Inverter will switch to IDLE state as soon as the battery SOC reaches its minimum value and PV power is 0W. If the sun comes out again it switches to Online state.

I am not sure if a new firmware than mine H3-8 (Master 2.11, Manager 1.85) supports changes on Standyparameters via inverterbuttons. It seems to freeze if I change something. I had to enable and change the limit to 20W via the Web-Interface.
